kairon/shared/chat/user_media.py

```
@@ -0,0 +1,189 @@
+            media_data.pop('bot')
+            media_data.pop('sender_id')
+            bucket = Utility.environment["storage"]["user_media"].get("bucket")
+            file_buffer = CloudUtility.download_file_to_memory(bucket, media_doc.filename)
+            download_name = f"{media_doc.bot}_{media_doc.sender_id}_{media_doc.filename}{media_doc.extension}"
+            if base64_encode:
+                file_buffer.seek(0)
+                encoded_string = base64.b64encode(file_buffer.read()).decode('utf-8')
+                return encoded_string, download_name
+            return file_buffer, download_name
+        except DoesNotExist:
+            raise AppException("Document not found")
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)